libc/utils/smhasher/smhasher.patch

```diff
@@ -0,0 +1,205 @@
+diff --git a/Hashes.h b/Hashes.h
```

Comment:
I also include a patch to `SMHasher` here to make it easier to verify the hash quality. Notice that there is ongoing work on [`SMHasher3`](https://gitlab.com/fwojcik/smhasher3/-/releases), but the author has not yet marked it as ready to use.

Reply:
I think it would be better to include a link to this in the commit description (or the PR description retained when merging) than as a .patch file in the tree.  Then we don't need this or libc/utils/smhasher/smhasher.txt additions.

Reply:
Updated in latest squashed commit.